tests/app/template/test_rest.py

```diff
@@ -261,47 +266,12 @@ def test_create_a_new_template_for_a_service_adds_folder_relationship(notify_db_
     )
     assert response.status_code == 201
 
-    stmt = select(Template).where(Template.name == 'my template')
+    stmt = select(Template).where(Template.name == 'my template 123')
```

Comment:
This could cause a failure when running tests in parallel.  Multiple templates can exist with the same name but different associated services.  Use `str(uuid4())` for the template name.

Reply:
Updated.